ironic.conf.j2

```diff
@@ -66,7 +66,8 @@ send_sensor_data_interval = 160
 bootloader = http://{{ env.IRONIC_URL_HOST }}:{{ env.HTTP_PORT }}/uefi_esp.img
 
 [database]
-connection = mysql+pymysql://ironic:{{ env.MARIADB_PASSWORD }}@localhost/ironic?charset=utf8
+connection = mysql+pymysql://ironic:{{ env.MARIADB_PASSWORD }}@127.0.0.1/ironic?charset=utf8&ssl=on&ssl_ca=/certs/ca/ironic/tls.crt
```

Comment:
you should parameterize this, we do not always deploy with ssl. You need to find out if mariadb uses ssl, and then add those parameters only in that case

Reply:
use an if else structure here for example

Reply:
The IRONIC_TLS_SETUP variable is tied to a file that is unrelated to mariadb, so you should tie this to the presence or not of the mariadb CA cert. You can create a new variable for that

Reply:
Done it! Thanks